sso/README.md

```diff
@@ -0,0 +1,32 @@
+## This module configures AWS Single Sign-On (SSO). AWS SSO makes it easy to centrally manage access to multiple AWS accounts and business applications and provide users with single sign-on access to all their assigned accounts and applications from one place. 
```

Comment:
can we move sso folder to have it as submodule in aws iam?

and also other depending submodules can be moved from /sso folder  as submodules, or they can stay as sub-sub modules under sso submodule if no separate usage from sso we suppose